### PR TITLE
Update newsletter and community call sign up to use Odoo survey

### PR DESCRIPTION
**Pull request by Ainali**
Exchange Google forms to the Odoo survey and added instructions
on how to use the survey for community call and newsletter sign up.

Addresses #813

-----
[View rendered activities/communication/index.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/communication/index.md)
[View rendered activities/communication/sign-up.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/communication/sign-up.md)
[View rendered activities/community-calls/index.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/community-calls/index.md)
[View rendered activities/live-streaming/post-production.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/live-streaming/post-production.md)
[View rendered activities/staff-meetings/event-debrief.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/staff-meetings/event-debrief.md)
[View rendered activities/standard-maintenance/preparing-community-call.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/standard-maintenance/preparing-community-call.md)
[View rendered activities/tool-management/odoo.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/activities/tool-management/odoo.md)
[View rendered index.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/index.md)
[View rendered organization/contact-details.md](https://github.com/publiccodenet/about/blob/ja-mail-collection/organization/contact-details.md)

### CODE REVIEW

File: activities/communication/sign-up.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+Type: guide
+Explains: how to manage sign ups for the newsletter and community calls
+---
+
+# Managing newsletter and community call sign up
+
+We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules that we use are Surveys and Contacts. In Odoo we have one survey called *Newsletter and community calls*, which is the one that configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).
```
Comment (demkodo) [inline activities/communication/sign-up.md:8]: I'd remove "that" after "the modules". I'd replace "one survey

Comment (ElenaFdR) [inline activities/communication/sign-up.md:8]:
```suggestion
We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules we use are Surveys and Contacts. In Odoo we have a survey called *Newsletter and community calls*, which configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).
```

File: activities/communication/sign-up.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+The link for this form can be shared and used wherever we feel that those we are communicating with may be looking for ways to stay in touch with us.
+
+## Update contacts
+
+When preparing for sending out [invites to community calls](../standard-maintenance/preparing-community-call.md) or the newsletter, first the contacts need to be updated. Check in the notes in the bottom of the survey when it was last updated. In the top right corner of the survey there is a button stating how many answers the survey has. Clicking it reveals all the persons that have signed up and their options. By selecting all the answers that was `Created on` later than in it was last updated and then the download button in the top left corner you get all the answers in one file. With the details from the file, update existing contacts or create new ones. Use the tags, ´Newsletter´, ´Community Call / Foundation', ´Community Call /Standard´ and ´SendValuable´ to store their preferences for being contacted.
```
Comment (ElenaFdR) [inline activities/communication/sign-up.md:14]:
```suggestion
When preparing to send out [invitations to community calls](../standard-maintenance/preparing-community-call.md) or the newsletter, first the contacts need to be updated. Check in the notes in the bottom of the survey when it was last updated. In the top right corner of the survey there is a button stating how many answers the survey has. Clicking it reveals all the persons that have signed up and their options. By selecting all the answers that were `Created on` later than when it was last updated and then the download button in the top left corner you get all the answers in one file. With the details from the file, update existing contacts or create new ones. Use the tags ´Newsletter´, ´Community Call / Foundation', ´Community Call /Standard´ and ´SendValuable´ to store their preferences for being contacted.
```

File: activities/staff-meetings/event-debrief.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Individually, develop high level notes on who you met, what you learnt and poten
 1. New contacts:
     * who are the interesting people we met?
```
Comment (demkodo) [inline activities/staff-meetings/event-debrief.md:20]: we've met

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
Small changes, then ready to go! Are you also updating the universal footer on the Jekyll theme?

**Comment by ElenaFdR** [comment]
Sorry, I've just spotted that that's already covered in the issue.

File: activities/communication/sign-up.md (hunk 1 above)
Comment (demkodo) [inline activities/communication/sign-up.md:8]:
```suggestion
We use [Odoo](../tool-management/odoo.md) to manage the sign up process. The modules that we use are Surveys and Contacts. In Odoo we have one survey called *Newsletter and community calls*, which is the one that configures the [sign up form](https://odoo.publiccode.net/survey/start/594b9243-c7e5-4bc1-8714-35137c971842).
```

**Review by ElenaFdR — APPROVED** [review]
This is good to go! An idea for a future iteration: add info somewhere that this approach keeps us GDPR compliant because we have a record showing that we have someone's contact details by their consent (#350).

**Pull request merged**